site/dat/docs/Nose.md

```diff
@@ -32,27 +32,99 @@ Nose executor supports building test script from the `requests` option of `scena
 generate a Python script that will be launched with `nose`.
 
 Supported features:
-  - select browser
+  - select browser Chrome, Firefox, Android-Chome (Appium), iOS-Safari (Appium) 
+  - local webdriver, local remote webdriver or remote webdriver (selenium grid or others like browserstack, saucelabs and similars)
```

Comment:
please remove browserstack and saucelabs as they are commercial companies mentioned